DUI3-DX/Converters/Rhino/Speckle.Converters.Rhino7/ToHost/Raw/SpecklePolylineRawToHostConversion.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Rhino.Collections;
+using Speckle.Converters.Common.Objects;
+
+namespace Speckle.Converters.Rhino7.ToHost.Raw;
+
+public class SpecklePolylineRawToHostConversion
+  : IRawConversion<SOG.Polyline, RG.Polyline>,
+    IRawConversion<SOG.Polyline, RG.PolylineCurve>
+{
+  private readonly IRawConversion<IList<double>, Point3dList> _pointListConverter;
+  private readonly IRawConversion<SOP.Interval, RG.Interval> _intervalConverter;
+
+  public SpecklePolylineRawToHostConversion(
+    IRawConversion<IList<double>, Point3dList> pointListConverter,
+    IRawConversion<SOP.Interval, RG.Interval> intervalConverter
+  )
+  {
+    _pointListConverter = pointListConverter;
+    _intervalConverter = intervalConverter;
+  }
+
+  public RG.Polyline RawConvert(SOG.Polyline target)
+  {
+    var points = _pointListConverter.RawConvert(target.value);
+
+    if (target.closed)
+    {
+      points.Add(points[0]);
+    }
+
+    var poly = new RG.Polyline(points);
+
+    return poly;
+  }
+
+  RG.PolylineCurve IRawConversion<SOG.Polyline, RG.PolylineCurve>.RawConvert(SOG.Polyline target)
```

Comment:
Yeah this again, to me it smells but not _sure_ it should...

Reply:
Not sure this is an issue since we're always injecting the Interface not the specific type, we are in fact doing this already when consuming it.